tests/test_errorcallback.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the error callback"""
+
+import unittest
+import unittest.mock
+
+import rtmidi
+
+class RtMidiTestCase(unittest.TestCase):
```

Comment:
Ah, missed it copy pasting.

Wait, the class is a test case? What are the functions then?